src/activities/menu/Menu.qml

```diff
@@ -122,6 +128,9 @@ ActivityBase {
         id: background
         source: menuActivity.url + "background.svg"
         sourceSize.width: parent.width
+        height: menuActivity.currentTag === "search" && ApplicationSettings.isVirtualKeyboard ?
+                    main.height - (((keyboard.keyHeight+keyboard.rowSpacing) * keyboard.rows)+keyboard.margin)
```

Comment:
in your case, you compute the height of the virtual keyboard. It should be a property of the virtualKeyboard.
Instead of ((keyboard.keyHeight+keyboard.rowSpacing) * keyboard.rows)+keyboard.margin), you should have something like keyboard.height